docs/analyzer/user_guide.md

```diff
@@ -427,6 +433,11 @@ optional arguments:
                         Read the compiler includes and target from the
                         specified file rather than invoke the compiler
                         executable.
+  --skip-gcc-fix-include
```

Comment:
I suggest to mark this flag as [DEPRECATED] so we can take it out in the next releases if nobody uses it.